include/swift/Sema/CSFix.h

```diff
@@ -843,17 +843,26 @@ class MarkGlobalActorFunction final : public ContextualMismatch {
 
 /// Introduce a '!' to force an optional unwrap.
 class ForceOptional final : public ContextualMismatch {
+  unsigned UnwrapCount;
+
   ForceOptional(ConstraintSystem &cs, Type fromType, Type toType,
                 ConstraintLocator *locator)
       : ContextualMismatch(cs, FixKind::ForceOptional, fromType, toType,
                            locator) {
     assert(fromType && "Base type must not be null");
     assert(fromType->getOptionalObjectType() &&
            "Unwrapped type must not be null");
+
+    auto fromOptionals = fromType->getOptionalityDepth();
+    auto toOptionals = toType->getOptionalityDepth();
+    assert(fromOptionals > toOptionals);
+
+    UnwrapCount = fromOptionals - toOptionals;
   }
 
 public:
   std::string getName() const override { return "force optional"; }
+  unsigned int getUnwrapCount() const { return UnwrapCount; }
```

Comment:
Nit: just `unsigned` is fine since we store `unsigned` or let's store `unsigned int` too.

Reply:
Fixed!